files/zh-tw/web/javascript/reference/global_objects/date/index.md

```diff
@@ -174,6 +174,6 @@ yourFunctionReturn = printElapsedTime(yourFunction);
 
 {{Compat}}
 
-\[1] 一些瀏覽器在解析日期時間會發生問題： [3/14/2012 blog from danvk Comparing FF/IE/Chrome on Parsing Date Strings](http://dygraphs.com/date-formats.html)
+\[1] 一些瀏覽器在解析日期時間會發生問題： [3/14/2012 blog from danvk Comparing FF/IE/Chrome on Parsing Date Strings](https://dygraphs.com/date-formats.html)
 
-\[2] [ISO8601 Date Format is not supported](<https://msdn.microsoft.com/en-us//library/ie/ff743760(v=vs.94).aspx>) in Internet Explorer 8, and other version can have [issues when parsing dates](http://dygraphs.com/date-formats.html)
+\[2] [ISO8601 Date Format is not supported](<https://learn.microsoft.com/en-us/>).aspx>) in Internet Explorer 8, and other version can have [issues when parsing dates](https://dygraphs.com/date-formats.html)
```

Comment:
the first link to msdn was malformatted. while the correct one redircts to MDN /en-US/docs/Web/JavaScript/Guide/Numbers_and_dates

```suggestion
\[2] [ISO8601 Date Format is not supported](<https://msdn.microsoft.com/en-us//library/ie/ff743760(v=vs.94).aspx>) in Internet Explorer 8, and other version can have [issues when parsing dates](https://dygraphs.com/date-formats.html)
```